.release-notes/4147.md

```diff
@@ -0,0 +1,14 @@
+## Change error message
+
+The error message when assigning to a field of a field in box functions was fixed to reflect that the function should be a fun ref. This affects code like the below:
+
+```pony
+actor Main
+        let my_test_class: TestClass= TestClass
+        new create(env: Env) =>
+                """placeholder"""
+        fun test() =>
+                my_test_class.x= 1
+```
```

Comment:
Any pony code in release notes should be formatted with our standard "2 spaces for indentation". Additionally it violates our coding standards in a number of other ways.

Reply:
i updated it to use two spaces, let me know about any other changes needed